en/docs/get-started/about-this-release.md

```diff
@@ -65,6 +65,7 @@ It is now available to download from [here](https://wso2.com/integration/integra
 
 ### New features
 
+- **[Generating Service Catalog Metadata Artifact for Older Workspaces]({{base_path}}/integrate/develop/generate-service-catalog-metadata.md)**
```

Comment:
Shall we add a small description as well under this?

Reply:
Fixed